### PR TITLE
Log in

### PR DESCRIPTION
**Pull request by krate27**
Made login page, plz review

### CODE REVIEW

File: src/Components/Auth/Login/login.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  const { loginAuth, token } = useAuth();
+  const [loginForm, setLoginForm] = useState({
+    email: "",
+    password: "",
+  });
+  useEffect(() => {
+    loginAuth(loginForm.email, loginForm.password)
+  }, [loginForm.email, loginForm.password]);
+  const navigate = useNavigate();
+  const location = useLocation();
```
Comment (deekshithmd) [inline src/Components/Auth/Login/login.js:15]: move these initializations to top, that will give good readability

File: src/Components/VideoCard/video-card.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import '../VideoCard/video-card.css';
+import { useNavigate } from 'react-router-dom';
+const VideoCard = ({ video }) => {
+  const navigate = useNavigate();
+    const {_id,title,catchName,uploaded} = video;
```
Comment (deekshithmd) [inline src/Components/VideoCard/video-card.js:5]: Suggestion: use some code formatters, that will make your code clean by providing necessary gaps and spaces

File: src/Pages/Video-Listing/videoListing.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    grid-template-columns: repeat(auto-fill,minmax(18rem,1fr));
+    grid-gap: 1rem;
+}
+.category-single{
+  color: var(--black-color);
+  font-size: 1.1rem;
+  cursor: pointer;
+  background-color: var(--body-bg-color);
+  border-radius: 2rem;
+  padding: 0.4rem 1rem;
```
Comment (deekshithmd) [inline src/Pages/Video-Listing/videoListing.css:12]: try to use px for values less than 1rem

**Review by deekshithmd — APPROVED** [review]

**Review by jeetbhalani15 — APPROVED** [review]
Code and folder looks good

**Pull request merged**